data-raw/covid_pmfs.R

```diff
@@ -7,20 +7,20 @@ params <- wwinference::get_params(
 
 # Put it all together
 generation_interval <- withr::with_seed(42, {
-  wwinference::simulate_double_censored_pmf(
+  wwinference:::simulate_double_censored_pmf(
     max = params$gt_max, meanlog = params$mu_gi,
     sdlog = params$sigma_gi, fun_dist = rlnorm, n = 5e6
-  ) |> wwinference::drop_first_and_renormalize()
+  ) |> wwinference:::drop_first_and_renormalize()
 })
 
-inc <- wwinference::make_incubation_period_pmf(
+inc <- wwinference:::make_incubation_period_pmf(
   params$backward_scale, params$backward_shape, params$r
 )
-sym_to_hosp <- wwinference::make_hospital_onset_delay_pmf(
+sym_to_hosp <- wwinference:::make_hospital_onset_delay_pmf(
   params$neg_binom_mu,
   params$neg_binom_size
 )
-inf_to_hosp <- wwinference::make_reporting_delay_pmf(inc, sym_to_hosp)
+inf_to_hosp <- wwinference:::make_reporting_delay_pmf(inc, sym_to_hosp)
```

Comment:
```suggestion
inf_to_hosp <- wwinference::make_reporting_delay_pmf(inc, sym_to_hosp)
```

Reply:
I think these triple colors were introduced erroneously! They would error if you tried to run them but doesn't look like you did.